OTEL_METRICS_GUIDE.md

```diff
@@ -0,0 +1,309 @@
+        KeyValue::new("route", uri.path().to_string()),
+    ]);
+
+    response
+}
+
+#[derive(serde::Serialize)]
+struct User {
+    id: u32,
+    name: String,
+}
+```
+
+### Actix Web Example
+
+Similar integration is available for Actix Web:
+
+```rust
+use actix_web::{App, HttpServer, web, HttpResponse};
+use opentelemetry_instrumentation_actix_web::{RequestMetrics, RequestTracing};
+use opentelemetry::{KeyValue, metrics::Counter};
+use std::sync::LazyLock;
+
+static API_CALLS: LazyLock<Counter<u64>> = LazyLock::new(|| {
+    logfire::u64_counter("api_calls_total")
+        .with_description("Total API calls")
+        .with_unit("{call}")
+        .build()
+});
+
+#[actix_web::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let shutdown_handler = logfire::configure()
+        .install_panic_handler()
+        .finish()?;
+
+    HttpServer::new(|| {
+        App::new()
+            // OpenTelemetry middleware for automatic tracing and metrics
+            .wrap(RequestTracing::new())
+            .wrap(RequestMetrics::default())
+            .route("/api/health", web::get().to(health_check))
+    })
+    .bind("127.0.0.1:3000")?
+    .run()
+    .await?;
+
+    shutdown_handler.shutdown()?;
+    Ok(())
+}
+
+async fn health_check() -> HttpResponse {
+    API_CALLS.add(1, &[KeyValue::new("endpoint", "health")]);
+    HttpResponse::Ok().json(serde_json::json!({"status": "healthy"}))
+}
+```
+
+## Observable Metrics
+
+For metrics that need to be sampled periodically rather than recorded on-demand:
+
+```rust
+use std::sync::LazyLock;
+use opentelemetry::metrics::ObservableGauge;
+
+static MEMORY_USAGE: LazyLock<ObservableGauge<u64>> = LazyLock::new(|| {
+    logfire::u64_observable_gauge("memory_usage_bytes")
+        .with_description("Current memory usage")
+        .with_unit("By")
+        .with_callback(|observer| {
+            // Get current memory usage (example)
+            let memory_usage = get_memory_usage();
+            observer.observe(memory_usage, &[]);
+        })
+        .build()
+});
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let shutdown_handler = logfire::configure().finish()?;
+
+    // Initialize the observable metric
+    LazyLock::force(&MEMORY_USAGE);
+
+    // Keep the application running to allow periodic observations
+    std::thread::sleep(std::time::Duration::from_secs(60));
+
+    shutdown_handler.shutdown()?;
+    Ok(())
+}
+
+fn get_memory_usage() -> u64 {
+    // Implementation to get actual memory usage
+    1024 * 1024 * 100 // Example: 100MB
+}
+```
+
+## Using Metrics in Logfire Dashboards
+
+Once your metrics are being exported to Logfire, you can create dashboards and alerts:
+
+### 1. **View Metrics in Logfire**
+
+- Navigate to your Logfire project dashboard
+- Go to the ["Explore"](https://logfire.pydantic.dev/docs/guides/web-ui/explore/) view
+- Run `select * from metrics` to view all incoming metrics from your application
```

Comment:
It would be really nice to have examples that matched some (or all of) our current built-in dashboards. In particular web frameworks, service and system metrics:

<img width="834" alt="image" src="https://github.com/user-attachments/assets/579635fa-749d-48fc-88fc-0c373ea01911" />